runtime/neumann/src/lib.rs

```diff
@@ -720,20 +738,23 @@ construct_runtime!(
 		Aura: pallet_aura::{Pallet, Storage, Config<T>} = 23,
 		AuraExt: cumulus_pallet_aura_ext::{Pallet, Storage, Config} = 24,
 
+		// Utilities
+		Valve: pallet_valve::{Pallet, Call, Config, Storage, Event} = 30,
```

Comment:
I am not sure if shifting these will force me to change the version
https://github.com/OAK-Foundation/OAK-blockchain/blob/master/runtime/neumann/src/lib.rs#L183

Reply:
We'll do a blanket bump before the next upgrade (polka client 0.9.16)